tests/conftest.py

```diff
@@ -11,10 +11,6 @@ def test_data_path():
 
 @pytest.fixture
 def local_file_collections(test_data_path):
-    with open(test_data_path / "users.csv", "r") as f:
-        or_users = f.read()
-    with open(test_data_path / "demographics.csv", 'r') as f:
-        or_dems = f.read()
```

Comment:
Let's go ahead and remove the text files as well located at `tests/data/users.csv` and `tests/data/demographics.csv`.